src/studio-home/card-item/index.tsx

```diff
@@ -24,7 +27,12 @@ interface BaseProps {
   rerunLink?: string | null;
   courseKey?: string;
   isLibraries?: boolean;
+  isMigrated?: boolean;
```

Comment:
Nit: This is always returned, right?

```suggestion
  isMigrated: boolean;
```

Reply:
This component is used in course and libraries v2 pages as well, so we need to keep it optional here.